apps/linode-marketplace-mysql/roles/post/tasks/main.yml

```
@@ -1,16 +1,22 @@
 ---
-  - name: Copy MOTD template to /etc/motd
-    template:
-      src: 'templates/motd.j2'
-      dest: '/etc/motd'
+# Install Addons if requested
+- name: add addons
+  import_role:
+    name: addons
+  when: add_ons != 'none'
```

```suggestion
  when: add_ons != ['none']
```
Otherwise this is always run.